yaml.properties

```diff
@@ -1,12 +1,19 @@
 category=Languages
 description=Allows the analysis of YAML documents
 homepageUrl=https://github.com/sbaudoin/sonar-yaml
-archivedVersions=1.4.3,1.5.0,1.5.1,1.5.2
-publicVersions=1.6.0
+archivedVersions=1.4.3,1.5.0,1.5.1,1.5.2,1.6.0
+publicVersions=1.7.0
 
 defaults.mavenGroupId=com.github.sbaudoin
 defaults.mavenArtifactId=sonar-yaml-plugin
 
+
+1.7.0.description=Allows rule customization from a local .yamllint file and fixes some documentation inconsistencies
+1.7.0.sqVersions=[7.9,LATEST]
```

Comment:
Support for anything <8.9 is now officially completely over. This range needs to start at 8.9.